src/Powercord/plugins/pc-codeblocks/style.scss

```
@@ -24,14 +24,8 @@
   }
 }
 
-.theme-light {
-  .powercord-codeblock-lang {
-    color: #777777a8 !important;
-  }
-}
-
 .powercord-codeblock-lang {
-  color: #ffffff;
+  color: var(--text-muted);
```

Uh, same case here. I'd personally use `--text-normal` which is very close to the traditional white for dark theme and a far more standing out black for light themed users.

```suggestion
  color: var(--text-normal);
```